roles/etcd/meta/main.yml

```diff
@@ -18,3 +18,4 @@ galaxy_info:
 dependencies:
 - { role: os_firewall }
 - { role: etcd_common }
+- { role: etcd_server_certificates }
```

Comment:
I think I like that idea better, so that other roles that would need/want an independent CA can piggyback off of it (such as the new nuage certificate bits.